CHANGES.rst

```diff
@@ -204,6 +218,8 @@ API changes
 - ``astropy.conftest.py``
 
 - ``astropy.constants``
+  - Changed constant names for solar/planetary conversion factors 
+    (R_sun -> R_Sun, R_earth -> Re_Earth, etc.)
```

Comment:
Strongly second the concerns about the name change, except for the 3rd point:
the existing R_earth, R_jup already refer to the equatorial radii. One could therefore add the polar values and link the old name to the equatorial one:
`R_earth = Re_earth` etc.
Although it could also be argued for defining a volume-equal mean radius
`R_earth = (Re_earth**2 * Rp_earth)**(1./3)`
but that would again create inconsistencies with many people's current usage.